package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "NODE_ENV=test mocha ./tests --timeout=10000 --exit"
```

Comment:
@niomwungeri-fabrice, May I know why this timeout argument???

Reply:
@espoirMur
by default, mocha runs the test in 5 seconds = 5000 milliseconds, as you know sometimes test that involves DB might take long to run, that's why as precaution Luc added another 5 seconds to avoid timeout warnings in future.